src/alfasim_sdk/hook_specs_gui.py

```diff
@@ -2,12 +2,11 @@
 from typing import Optional
 from typing import Union
 
-from pluggy import HookspecMarker
-
 from alfasim_sdk.context import Context
 from alfasim_sdk.status import ErrorMessage
 from alfasim_sdk.status import WarningMessage
 from alfasim_sdk.variables import SecondaryVariable
+from pluggy import HookspecMarker
```

Comment:
Isn't this a third party module?
The right place is over there where it came from =)

Reply:
The `pre-commit` was wrongly configured, the parameter `--application-directories=.:src` was missing.
The next commit fix this issue.